.github/workflows/mongodb_settings.py

```diff
@@ -15,13 +15,13 @@
 else:
     DATABASES = {
         "default": {
-            "ENGINE": "django_mongodb_backend",
+            "ENGINE": "django_mongodb_backend_gis",
```

Comment:
Can this be `django_mongodb_backend.gis` ? I cannot with the `_gis` on the already too long `django_mongodb_backend`. Also similar to `django.contrib.gis`.